app/models/order.rb

```diff
@@ -0,0 +1,36 @@
+class Order < ActiveRecord::Base
+  # DB relationships
+  has_many :order_items
+
+  # validations helper regex
+  # email regex from: http://rails-3-2.railstutorial.org/book/modeling_users#code-validates_format_of_email
+  # VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-]+(\.[a-z\d\-]+)*\.[a-z]+\z/i
+  VALID_STATUS_REGEX = /(pending)|(paid)|(complete)|(cancelled)/
+
+  # data validations
+  validates :status, presence: true, format: { with: VALID_STATUS_REGEX }
```

Comment:
It might be simpler to use the built in [inclusion](http://guides.rubyonrails.org/active_record_validations.html#inclusion) validation.